### ISSUE TITLE
Release 1.0?

### ISSUE THREAD

**Issue by curran** [share/sharedb-mongo#114]
It feels a bit odd that the latest version is marked as beta.

**Comment by alecgibson** [comment]
Yeah. There's probably a couple of things we should tidy up, like at least move to string error codes like [we did with `sharedb`](https://github.com/share/sharedb/pull/309).

We should probably also generally do an audit of the open issues and see if any of them should be marked as needed for v1.0. For example, I still think we should have a bit of a think on indexes as raised [here](https://github.com/share/sharedb-mongo/issues/94) and [here](https://github.com/share/sharedb-mongo/issues/66).

**Comment by curran** [comment]
Very nice! Thanks for taking a look here.
